core/types/transaction_signing.go

```diff
@@ -65,7 +65,7 @@ func MakeSigner(config *params.ChainConfig, blockNumber *big.Int, blockTime uint
 // have the current block number available, use MakeSigner instead.
 func LatestSigner(config *params.ChainConfig) Signer {
```

Comment:
There's also `func LatestSignerForChainID` which always returns `NewCancunSigner` for non-nil chain ids. Do we want to leave this because the cancun signer is also fine for op networks?

In general, I am wondering, how important is it that we return the `LondonSigner` for op chains instead of just the cancun signer? The networks doesn't accept blob txs, so if someone uses the cancun signer to sign a blob tx, the transaction will just be rejected as invalid. So couldn't we reduce the diff by just using the cancun signer also for op chains?

Reply:
The signer is used in the state-transition and other parts, to validate which tx can be included and which cannot. Specifically, the `signer  = types.MakeSigner(p.config, header.Number, header.Time)` in the state-transition. The other signers are for utils / peripherals.

Reply:
With the RPC, tx-pool and state-transition covered I think we can allow these non-config-initiated signers to just return the latest signer.